pkg/operator/reports_test.go

```diff
@@ -92,3 +101,154 @@ func TestGetNextReportPeriod(t *testing.T) {
 		})
 	}
 }
+
+func TestIsReportRunning(t *testing.T) {
+	const (
+		testNamespace  = "default"
+		testReportName = "test-report"
+		testQueryName  = "test-query"
+	)
+
+	schedule := &metering.ReportSchedule{
+		Period: metering.ReportPeriodCron,
+		Cron:   &metering.ReportScheduleCron{Expression: "5 4 * * *"},
+	}
+
+	reportStart := &time.Time{}
+	reportEndTmp := reportStart.AddDate(0, 1, 0)
+	reportEnd := &reportEndTmp
+
+	testTable := []struct {
+		name       string
+		report     *metering.Report
+		expectFlag bool
```

Comment:
This would be clearer as `expectRunning`

Reply:
I also changed the bool `expectFlag` -> `expectErr` in the second test function